docs/questionnaire-yaml.md

```diff
@@ -0,0 +1,61 @@
+    6. yellow: Required string message to display in reports for the yellow risk level.
+    7. green: Required string message to display in reports for the green risk level.
+    8. unknown: Required string message to display in reports for the unknown risk level.
+5. sections: Required list of sections that the questionnaire will include.
+    1. name: Name is the required string to be displayed for the section.
+    1. order: Required int order in which the question should appear in the section.
+    1. comment: Optional string to describe the section.
+    1. questions: Required list of questions that belong to the section. 
+        1. order: Required int order in which the question should appear in the section.
+        1. text: Required string of the question to be asked.
+        1. explanation: Optional string of additional explanations for the question.
+        2. includeFor: Optional list that defines a question should be displayed if any of the tags included in the list is present in the target application or archetype.
+            1. category: Required string category of the target tag.
+            2. tag: Required string for the target tag.
+        3. excludeFor: Optional list defines that a question should be skipped if any of the tags included in the list is present in the target application or archetype.
+            1. category: Required string category of the target tag.
+            2. tag: Required string for the target tag.
+        4. answers: Required list of answers for the given question. 
+            1. order: Required int order in which the question should appear in the section.
+            1. text:  Required string the actual answer for the question.
+            2. risk: Required to be one of red, yellow, green, or unknown. The risk level the current answer implies.
+            3. rationale:  Optional string explaining the justification for the answer being considered a risk.
+            4. mitigation: Optional string for an explanation of the potential mitigation strategy for the risk implied by this answer.
+            5. applyTags: Optional list that defines a list of tags to be automatically applied to the assessed application or archetype if this answer is selected.
+                1. category: Required string category of the target tag.
+                2. tag: Required string for the target tag.
+            6. autoAnswerFor: Optional list defines a list of tags that will lead to this answer being automatically selected when the application or archetype is assessed. 
+                1. category: Required string category of the target tag.
+                2. tag: Required string for the target tag.
+
+#### Notes
+
+1. Anything with the word **required** must be filed out. Otherwise, the yaml will not validate on upload.
+2. Each subsection defines a new struct/object in yaml. For instance
+
+```yaml
+...
+name: Testing
+thresholds: 
+    red: 30
+    yellow: 45
+    unknown: 5
+...
+```
```

Comment:
You could use a NOTE admonition to highlight

```suggestion
> [!NOTE]
>
> 1. Anything with the word **required** must be filed out. Otherwise, the YAML will not validate on upload.
> 2. Each subsection defines a new struct/object in yaml. For instance
> ```yaml
> ...
> name: Testing
> thresholds: 
>    red: 30
>    yellow: 45
>    unknown: 5
> ...
> ```
```

Reply:
Does that note syntax work with markdown or only with asidoc?

Reply:
It is markdown syntax for an admonition:

![image](https://github.com/konveyor/tackle2-hub/assets/106804821/a1a2ef69-7759-496e-a217-b280d6a1879e)

AsciiDoc admonitions are silently different. I miss markdown so much

Reply:
Wow! never knew this will change!

Reply:
updated